lib/tree-view.coffee

```diff
@@ -205,19 +219,19 @@ class TreeView extends View
       @show()
 
   entryClicked: (e) ->
-    entry = e.currentTarget
+    entry = e.target.closest('.entry')
     isRecursive = e.altKey or false
     @selectEntry(entry)
-    if entry instanceof DirectoryView
+    if entry.classList.contains('directory')
```

Comment:
This raises an exception if the user clicks on the tree-view's background, rather than a specific entry:

~~~
tree-view.coffee [sm]:225
Uncaught TypeError: Cannot read property 'classList' of null
	module.exports.TreeView.entryClicked  @ tree-view.coffee [sm]:225
	(anonymous function)                  @ tree-view.coffee [sm]:100
~~~